### ISSUE TITLE
JS Error on checkout when shipping form doesn't validate

### ISSUE THREAD

**Issue by kweij** [dhlparcel/plugin-magento2-release#20]
In the checkout I got the following error when any field on the shipping form didn't validate (an address field for example):

```
Uncaught TypeError: Cannot read property 'map' of undefined
    at UiClass._delegate (collection.js:318)
    at UiClass.delegate (collection.js:305)
    at collection.js:328
    at Function._.map._.collect (underscore.js:164)
    at Function.map (observable_array.js:50)
    at UiClass._delegate (collection.js:318)
    at UiClass.delegate (collection.js:305)
    at collection.js:328
    at Function._.map._.collect (underscore.js:164)
    at Function.map (observable_array.js:50)
```

dhlparcel/magento2-plugin version: 1.0.12
Magento version: 2.3.4-p2

I've done some investigating and found that the shipping form is validated on saving. This happens in `vendor/magento/module-ui/view/base/web/js/form/form.js`, method `save`. If it's not valid, it triggers the method `this.focusInvalid();` which in turn starts with the satement `var invalidField = _.find(this.delegate('checkInvalid'));`.

Here the `this.delegate('checkInvalid')` parts fails. The `delegate` method can be found in `vendor/magento/module-ui/view/base/web/js/lib/core/collection.js` and it triggers `this._delegate(args)`. That `_delegate` method runs throught all elements of the collection component with the statement `result = this.elems.map(function (elem) {`.

Here the `this.elems` part fails because `elems` was `undefined`. The components on which `this.elems` turned out to be `undefined` were `DHLParcel_Shipping/js/view/deliveryoptions-info` and `DHLParcel_Shipping/js/view/servicepoint-info`.

The reason that these components didn't have `elems` was because the `initObservable` method is overwritten _without_ calling `this._super();`. The collection component (the super class of these two classes) instantiates `this.elems`, but that didn't occur:

https://github.com/dhlparcel/plugin-magento2-release/blob/2d5f9c71c51301aea28409366ce3c7b15738a559/view/frontend/web/js/view/deliveryoptions-info.js#L34

https://github.com/dhlparcel/plugin-magento2-release/blob/2d5f9c71c51301aea28409366ce3c7b15738a559/view/frontend/web/js/view/servicepoint-info.js#L14

Here's the pull-request:
https://github.com/dhlparcel/plugin-magento2-release/pull/19

**Comment by roerlemans** [comment]
Thanks for your input! We have this fixed in our next release, thanks for your PR!

**Comment by roerlemans** [comment]
This bug is fixed in v1.0.16:
https://github.com/dhlparcel/plugin-magento2-release/releases/tag/v1.0.16
